agent/__init__.py

```diff
@@ -42,11 +47,8 @@
 
 # Conditional handling for if we're running
 # inside a Snap.
-if CONFINEMENT == Confinement.SNAP:
-    CONFIG_PATH = CERT_PATH = os.getenv('SNAP_DATA')
-else:
-    CERT_PATH = os.getenv('CERT_PATH', '/opt/wott/certs')
-    CONFIG_PATH = os.getenv('CONFIG_PATH', '/opt/wott')
+CERT_PATH = os.getenv('CERT_PATH', '/opt/wott/certs')
```

Comment:
Is this path really writeable inside the snap?

Reply:
Yes, because it's mapped. See WoTTsecurity/wott-agent-snap#4

Reply:
I don't think this is the right way of doing it. I think it needs to be in `SNAP_DATA` in order for the Snap to share it with other Snaps using Plugs.

Reply:
If so, it needs to be in `SNAP_COMMON` and have a `content` interface declared.
This code doesn't need to be changed, we can change the mapping in snapcraft.yaml.

Reply:
So everything that we might need to share (such as keys, certs and credentials) need to be in a storage that can be shared. Lockfile etc will never be used by another app, so we can put them elsewhere. 

Also, keep in mind that in a perfect world, we'll move the keys to a TPM, but that's another story.

Reply:
> So everything that we might need to share (such as keys, ...

Don't you think we shouldn't share keys?

> Lockfile etc will never be used by another app, so we can put them elsewhere.

That's why I was initially storing lockfiles in /var/lock, but you told me otherwise. Now that I can test snaps myself I'll put them in /var/lock again and see if it works out.

Reply:
Sorry for the misunderstanding.